### PR TITLE
Dashboard org cards

### PR DESCRIPTION
**Pull request by am5815**
Updated organization card to reflect card.
NOTE: The thumbnail right now is UC DAVIS, this will be updated to reflect correct organization.

### CODE REVIEW

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -7,9 +7,11 @@ export default function Login() {
     const navigate = useNavigate();
     const { isAuthenticated } = useContext(AuthContext);
     const projectId = import.meta.env.VITE_PROJECT_ID;
+    const loginUrl = `https://test-auth.sail.codes?projectId=${projectId}&redirectUrl=${encodeURIComponent(window.location.origin + '/auth/callback')}`;
```
Comment (cbolles) [inline src/pages/Login.tsx:10]: Shouldn't have a hard coded value here

**Review by cbolles — APPROVED** [review]

**Review by cbolles — CHANGES_REQUESTED** [review]
Just need to remove the hard coded value

File: src/pages/Dashboard.tsx [hunk 2]
```diff
@@ -16,16 +17,25 @@ export default function Dashboard() {
...
+        // card for each organization that show cases organization name and thumbnail image
         return organizations.map((organization: Organization) => (
-            <div key={organization.name}>
-                <div>
-                    <IconButton aria-label="delete" size="large" onClick={() => canClick ? routeToOrganization(organization) : alert("Contact Administrator for Org to request Access")}>
-                        <FolderOpenOutlinedIcon fontSize='large' />
-                        {organization.name}
-                    </IconButton>
-                </div>
+            <div key={organization.name} style={{ margin: 20 }}>
```
Comment (gregfrasco) [inline src/pages/Dashboard.tsx:24]: Should we make this its own component?

File: src/pages/Dashboard.tsx [hunk 3]
```diff
@@ -85,33 +84,41 @@ export default function Dashboard() {
...
+            {
+                adminOrganizations.length > 0 ?
+                <>
+                    <Typography variant='h3'>My Organizations</Typography>
+                    <div style={{ display: 'flex' }}>
+                        {renderOrganizations(adminOrganizations, true, 'Admin')}
+                    </div>
+                </>
+                : null
+            }
```
Comment (gregfrasco) [inline src/pages/Dashboard.tsx:100]: nit: could be refactor to this
```suggestion
            { adminOrganizations.length && 
               <>
                    <Typography variant='h3'>My Organizations</Typography>
                    <div style={{ display: 'flex' }}>
                        {renderOrganizations(adminOrganizations, true, 'Admin')}
                    </div>
                </>
            }
```

**Review by gregfrasco — APPROVED** [review]
Minor changes

**Review by cbolles — APPROVED** [review]

**Pull request merged**